lib/knock.rb

```diff
@@ -2,7 +2,7 @@
 
 module Knock
   mattr_accessor :token_lifetime
-  self.token_lifetime = 1.day
+  self.token_lifetime = { default: 1.day }
```

Comment:
For compatibility, I think it's better to continue allowing the numeric values here like before...

The key must be the entity name that Knock is being used for (user, admin, manager, etc...), and we need to update it in README, adding an example.